chia/plotting/create_plots.py

```diff
@@ -210,11 +210,22 @@ async def create_plots(
 
         if args.plotid is not None:
             log.info(f"Debug plot ID: {args.plotid}")
-            plot_id = bytes32(bytes.fromhex(args.plotid))
+            # Check if args.memo is of type bytes and convert it to a string if so
+            if isinstance(args.plotid, bytes):
+                plot_str = args.plotid.hex()  # Convert bytes to hex string
+            else:
+                plot_str = args.plotid
+            plot_id = bytes32.fromhex(plot_str)
 
         if args.memo is not None:
+            print(f"Type of args.memo: {type(args.memo)}")  # Should print: <class 'str'>
```

Comment:
this seems like a debugging print that should be removed

Reply:
I removed the debugging print line and fixed the bytes32. Did we want to leave in the changes for the type check or try to fix chia plotters?